rt/rs/description-swagger/src/main/java/org/apache/cxf/jaxrs/swagger/openapi/SwaggerToOpenApiConversionFilter.java

```diff
@@ -55,7 +57,8 @@ public void filter(ContainerRequestContext reqCtx) throws IOException {
     @Override
     public void filter(ContainerRequestContext reqCtx, ContainerResponseContext respCtx) throws IOException {
         if (Boolean.TRUE == reqCtx.getProperty(OPEN_API_PROPERTY)) {
-            String swaggerJson = (String)respCtx.getEntity();
+            String swaggerJson = respCtx.getEntity() instanceof String ? (String)respCtx.getEntity()
```

Comment:
Minor, `String swaggerJson = Json.mapper().writeValueAsString(respCtx.getEntity());` should do it.

Reply:
I was trying to maintain the exact functionality in a 1.5.17 world if I could.

Reply:
It is exactly the same: the mapper does nothing for `String` (returns the same string back) but convert to JSON any object (same is done with `?:`)

Reply:
@jwcarman Please disregard my comment, the behavior is not identical (I thought it was).